Quick Start 文档缺少关键环境前置与常见报错说明（pnpm / tsc / zod）

## 背景
新用户按 README 的 Quick Start 从零克隆并启动项目时，遇到多个基础环境相关问题。当前 Quick Start 步骤不够自洽，容易在首跑阶段失败。

## 复现步骤
```bash
git clone https://github.com/zts212653/clowder-ai.git
cd clowder-ai
cp .env.example .env
pnpm install
pnpm start
```

## 实际问题
1. `pnpm` 安装前置不够显眼
   - README 的 Quick Start 直接使用 `pnpm`，但没有在该段落明确先安装 pnpm。
2. `tsc` 相关失败缺少排查指引
   - 在首跑构建链路里会触发 TypeScript 编译；若环境异常，用户会看到 `tsc` 相关错误（例如 command not found）。
3. `import { z } from 'zod'` 相关编译/解析报错缺少说明
   - 部分新环境会出现 `zod` 解析失败（例如 `Cannot find module 'zod'` 或对应类型声明问题），Quick Start 无 troubleshooting 指引。

## 期望行为
- 在 README Quick Start 附近补充明确前置：
  - Node.js 版本要求
  - pnpm 版本要求 + 安装命令（例如 `npm install -g pnpm`）
- 补一段首跑排障（可链接到 SETUP.md）：
  - `pnpm` not found
  - `tsc` not found
  - `zod` module/type resolution 错误
- 说明推荐命令顺序（确保在仓库根目录执行）：
  - `pnpm install`
  - `pnpm start`

## 参考位置
- README Quick Start: `README.md`（中英文两段）
- Setup Prerequisites: `SETUP.md`（已有更完整前置）

建议将 SETUP 的关键前置精简同步到 README Quick Start，以降低首次上手失败率。

--- a/README.md
+++ b/README.md
@@ -143,6 +143,8 @@ You don't need to be a developer. You need to know what you want — and who you
 
 > CVO Bootcamp coming soon — a guided onboarding where your AI team walks you through a complete feature lifecycle.
 
+**Prerequisites:** Node.js >= 20, pnpm >= 9 (`npm install -g pnpm`). Full details in [SETUP.md](SETUP.md).
+
 ```bash
 git clone https://github.com/zts212653/clowder-ai.git
 cd clowder-ai
@@ -495,6 +497,8 @@ Clowder 不只是一个编程平台。你的 AI 团队还能：
 ## 快速开始
 
 > CVO 训练营即将推出 — AI 团队亲自带你走完一个完整的 feature 生命周期。
+
+**前置要求：** Node.js >= 20、pnpm >= 9（`npm install -g pnpm`）。完整说明见 [SETUP.md](SETUP.md)。
 
 ```bash
 git clone https://github.com/zts212653/clowder-ai.git

--- a/SETUP.md
+++ b/SETUP.md
@@ -217,6 +217,19 @@ pnpm redis:user:backup  # Manual backup
 - Make sure `NEXT_PUBLIC_API_URL=http://localhost:3004` is set
 - API must be running before frontend loads
 
+**`pnpm` not found?**
+- Install pnpm: `npm install -g pnpm` (requires Node.js >= 20)
+- Verify: `pnpm --version` should show >= 9.0.0
+- If `npm` is also missing, install Node.js first from [nodejs.org](https://nodejs.org/)
+
+**`tsc` not found or TypeScript compile errors?**
+- Run `pnpm install` first — TypeScript is a project dependency, no global install needed
+- If errors persist: `rm -rf node_modules && pnpm install`
+
+**`Cannot find module 'zod'` or type resolution errors?**
+- Same fix: `rm -rf node_modules && pnpm install`
+- Check Node.js version: `node -v` (must be >= 20)
+
 ---
 
 <a id="中文"></a>
@@ -431,3 +444,16 @@ pnpm redis:user:backup  # 手动备份
 **前端连不上 API？**
 - 确认设了 `NEXT_PUBLIC_API_URL=http://localhost:3004`
 - API 必须在前端加载前启动
+
+**找不到 `pnpm`？**
+- 安装：`npm install -g pnpm`（需要 Node.js >= 20）
+- 验证：`pnpm --version` 应显示 >= 9.0.0
+- 如果 `npm` 也没有，先从 [nodejs.org](https://nodejs.org/) 安装 Node.js
+
+**`tsc` 找不到或 TypeScript 编译报错？**
+- 先运行 `pnpm install` — TypeScript 是项目依赖，不需要全局安装
+- 仍然报错：`rm -rf node_modules && pnpm install`
+
+**`Cannot find module 'zod'` 或类型解析报错？**
+- 同样的修复：`rm -rf node_modules && pnpm install`
+- 检查 Node.js 版本：`node -v`（必须 >= 20）

